services/records/record_templates/schemas.go

```diff
@@ -0,0 +1,161 @@
+package record_templates
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var validRecordTemplatesConstraintTypes = []string{"UNKNOWN", "PASSED", "FAILED"}
+
+var recordTemplateSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+		ForceNew: true,
+	},
+	"description": {
+		Type:     schema.TypeString,
+		Optional: true,
+		ForceNew: true,
+	},
+	"state": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"start_date_time": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"stop_date_time": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"stream_id": {
+		Type:     schema.TypeString,
+		Computed: true,
```

Comment:
it's an input so it should be
```suggestion
		Optional: true,
```